rules/opentitan_test.bzl

```diff
@@ -452,3 +454,138 @@ def opentitan_functest(
             # For more see https://bazel.build/reference/be/general#test_suite.tags
         ],
     )
+
+def _opentitan_sram_fpga_cw310_test(ctx):
+    test_script = '''#!/usr/bin/env bash
+set -e
+
+COLOR_RED='\\x1b[0;31m'
+COLOR_GREEN='\\x1b[0;32m'
+COLOR_PURPLE='\\x1b[0;35m'
+
+function prefix_lines() {{
+    LABEL="$1"
+    COLOR="$2"
+    COLOR_RESET='\\x1b[m'
+    sed -Eu "s/(.*)/$COLOR[$LABEL]$COLOR_RESET \\1/"
+}}
+
+set -x
+
+./opentitantool load-bitstream --rom-kind=TestRom test_rom.bit
```

Comment:
nit: we will need to support different bitstreams to be able to run tests using different OTP configurations.

Reply:
Ah ok, I'll have to figure out an ergonomic way to pass the ROM kind to opentitantool.

Reply:
SG, this can also be done in a follow up PR if you want.